.github/ISSUE_TEMPLATE.md

```diff
@@ -1,40 +1,6 @@
-<!--- Provide a general summary of the issue in the Title above -->
+Thank you for making an issue.
+If you are submitting a bug report, it will help us if you include the following information:
 
-> _This template is rather extensive. Fill out all that you can, if are a new contributor or you're unsure about any section, leave it unchanged and a reviewer will help you_ :smile:. _This template is simply a tool to help everyone remember the BioJulia guidelines, if you feel anything in this template is not relevant, simply delete it._
+- Your version of Julia and {THIS PACKAGE}
```

Comment:
Maybe better to write "Julia and all used packages", both so we don't forget to replace THIS PACKAGE and so users remember other deps